docusaurus/docs/this-document.md

```diff
@@ -0,0 +1,24 @@
+---
+id: this-document
+title: This document
+sidebar_label: This document
+---
+
+This document is the default documentation for [`next-with-moxy`](https://github.com/moxystudio/next-with-moxy) projects.
+
+**Keep in mind**, if you have received a project developed with [`next-with-moxy`](https://github.com/moxystudio/next-with-moxy), you have received a version of this documentation tailored to your project.
```

Comment:
Instead of "have received", perhaps "might have received"? We've had conversations about how this might not always be the case, but i think in either case this part should not imply we always give this document, even if in practice we do.

Reply:
That would make it:

> **Keep in mind**, if you might have received ...

That's a bit confusing. I think it's better to assume this document will always be delivered.